### DISCUSSION TITLE
Is there a way to remove the "arbitrary checks" for width/height?

### DISCUSSION THREAD

**Discussion by cookmscott** [libvips/libvips#3215] [Q&A] [answered]
I'm dealing with an image that goes past the width defined in:
https://github.com/libvips/libvips/blob/master/libvips/foreign/tiff2vips.c#L2843-L2851

Is there a simple way to exclude this check without removing and recompiling?

**Comment by jcupitt** [comment, marked as answer]
Hi @cookmscott,

More than 10m pixels on an axis? Wow!

It's supposed to be a sanity check -- we didn't think anyone would go higher. Yes, you'd need to change that `#define` and recompile.

https://github.com/libvips/libvips/blob/master/libvips/include/vips/image.h#L57-L64

What would you prefer as an upper bound?

**Comment by cookmscott** [comment]
The image doesn't have a ton of data, but it is a very long strip of sequence data, 89MP in horizontal direction. Please do not change anything on my behalf, I know what I'm working with is a bit crazy :) 

Thanks for the quick response.
